apb_devel/writing/reference.adoc

```diff
@@ -557,6 +557,180 @@ etherpad_db_host: "{{ lookup('env','ETHERPAD_DB_HOST') | default('mariadb', true
+endif::[]
+ifdef::openshift-origin[]
+$ oc get route -n ansible-service-broker
+endif::[]
+
+NAME       HOST/PORT                                           PATH      SERVICES   PORT        TERMINATION   WILDCARD
+asb-1338   asb-1338-ansible-service-broker.172.17.0.1.nip.io             asb        port-1338   reencrypt     None
+
+$ curl -H "Authorization: Bearer $(oc whoami -t)" -k -X POST https://asb-1338-ansible-service-broker.172.17.0.1.nip.io/ansible-service-broker/v2/bootstrap                                 
+{
+  "spec_count": 38,
+  "image_count": 109
+}
+----
++
+[NOTE]
+====
+The `oc whoami -t` command should return a token and the authenticated user must
+have permissions as described in
+xref:../../apb_devel/cli_tooling.adoc#apb-devel-cli-install-prereqs-access-permissions[Access Permissions].
+====
+
+. Verify the new APB exists in the OAB. This is normally the functionality of the
+`apb list` command. If you do not have the `apb` tooling installed, you can use
+the route gathered from the previous step and run:
++
+----
+$ curl -H "Authorization: Bearer $(oc whoami -t)" -k https://asb-1338-ansible-service-broker.172.17.0.1.nip.io/ansible-service-broker/v2/catalog
+----
++
+You should see a list of all bootstrapped specs and one that is labeled
+`localregistry-<bundle_name>`. Use `|grep <bundle_name>` to help find it, since
+the output is in JSON.
+
+[[apb-devel-writing-ref-remote-clusters-running-apbs]]
+=== Running APBs
+
+Because of the limitations when working with remote clusters, it may be desired
+for users to want the same functionality as the `apb run` command without having
+to rely on the `apb push` command being successful. This is because `apb run`
+implicitly performs `apb push` first before attempting to provision the
+application. 
+
+In order to work around this:
+
+. First, follow the steps described in
+xref:apb-devel-writing-ref-remote-clusters-pushing-apbs[Pushing APBs] to push
+your image onto the internal {product-title} registry. After the image exists,
+you should be able to see it with:
++
+----
+$ oc get images | grep <bundle_name>
+
+sha256:bfaa73a5e15bf90faec343c7d5f8cc4f952987afdbc3f11a24c54c037528d2ed   172.30.1.1:5000/openshift/<bundle_name>@sha256:bfaa73a5e15bf90faec343c7d5f8cc4f952987afdbc3f11a24c54c037528d2ed
+----
+
+. To provision, use the `oc run` command to launch the APB:
++
+----
+$ oc new-project <target_namespace>
+$ oc create serviceaccount apb
+$ oc create rolebinding apb --clusterrole=admin --serviceaccount=<target_namespace>:apb
+$ oc run <pod_name> \
+      --env="POD_NAME=<pod_name>" \
+      --env="POD_NAMESPACE=<target_namespace>" \
+      --image=<pull_spec> \ <1>
+      --restart=Never \
+      --attach=true \
+      --serviceaccount=apb \
+      -- <action> -e namespace=<target_namespace> -e cluster=$CLUSTER
+----
+<1> Specify the pull spec for the image shown when running `oc get images` from the
+previous step, since the registry will determine the FQDN.
+
```

Comment:
fully qualified domain name (FQDN) 

^unless it's not already defined somewhere on the page